simplefs/simplefs.go

```diff
@@ -1885,8 +1886,7 @@ func (k *SimpleFS) TlfHandleChange(_ context.Context, _ *libkbfs.TlfHandle) {
 // SimpleFSSuppressNotifications suppresses FSEvent and FSSyncEvent
 // notifications.
 func (k *SimpleFS) SimpleFSSuppressNotifications(ctx context.Context, suppressDurationSec int) error {
-	k.config.Reporter().SuppressNotifications(ctx, time.Duration(suppressDurationSec)*time.Second)
-	return nil
+	return errors.New("this rpc is deprecated")
```

Comment:
I'll have a separate set of PRs to actually remove this from the protocol after these are merged.